liquidator/liquidator.js

```diff
@@ -103,25 +103,88 @@ class Liquidator {
   queryAndWithdrawRewards = async () => {
     Logger.info({
       at: "liquidator",
-      message: "Checking for liquidations",
-      inputPrice: priceFeed
+      message: "Checking for expired and disputed liquidations to withdraw rewards from"
     });
 
     // Update the client to get the latest information.
     await this.empClient._update();
 
-    // TODO: Just showing an example of how I want to use the client:
-    // Get expired liquidations from the client.
-    const expiredLiquidations = this.empClient.getExpiredLiquidations();
-    // TODO: 
-    // - Withdraw rewards
-    // - Check that amount of rewards was correct.
+    // Update the gasEstimator to get the latest gas price data.
+    // If the client has a data point in the last 60 seconds returns immediately.
+    await this.gasEstimator._update();
 
-    // Get disputed liquidations from the client.
+    // All of the liquidations that we could withdraw rewards from are drawn from the pool of
+    // expired and disputed liquidations.
+    const expiredLiquidations = this.empClient.getExpiredLiquidations();
     const disputedLiquidations = this.empClient.getDisputedLiquidations();
-    // TODO: 
-    // - Withdraw rewards
-    // - Check whether it was a successful or failed dispute and double check that the reward amount was correct
+    const potentialWithdrawableLiquidations = expiredLiquidations.concat(disputedLiquidations);
```

Comment:
The below code chunk (122-162) is repeated for both expired and disputed liquidations, so consolidated them by concating their returned arrays